### PR TITLE
updated play button for tease

### PR DESCRIPTION
**Pull request by okeinom**

### CODE REVIEW

**Comment by okeinom** [comment]
https://audioverse7.atlassian.net/jira/software/projects/AV/boards/3?assignee=712020%3Afbf78bba-2174-45a3-bda3-9bd364fbc1ee&selectedIssue=AV-1076

File: src/components/molecules/teaseRecording.module.scss [hunk 1]
```diff
@@ -40,19 +62,23 @@ $container-padding: 16px;
 	display: flex;
 	justify-content: space-between;
 	align-items: flex-start;
-	margin-bottom: 16px;
+	overflow: hidden;
+	margin-bottom: 10px;
+	padding-bottom: 4px;
```
Comment (alangumer) [inline src/components/molecules/teaseRecording.module.scss:67]: On small screens, when the title is longer than two lines, the top of the third line is shown, one way to fix it is to reduce the padding to 2px.

**Review by alangumer — CHANGES_REQUESTED** [review]
Fix the translations for time left.

File: src/components/atoms/playButtonCurrentLockup.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+type PlayButtonCurrentProps = {
+	isPlaying: boolean;
+	isCurrentTrack: boolean;
+	position: number;
+	iconActiveColor: BaseColors;
+	trackColor: BaseColors;
+	bufferedProgress: number;
+};
+
+const PlayButtonCurrentLockup: React.FC<PlayButtonCurrentProps> = ({
```
[the remark this answers is not in the document]
Comment (okeinom) [reply src/components/atoms/playButtonCurrentLockup.tsx:16]: removed

File: src/components/molecules/buttonPlayCircle.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+				})}
+			>
+				{isCurrentTrack ? (
+					<>
+						<PlayButtonCurrentLockup
+							isPlaying={session.isPlaying}
+							position={session.progress}
+							iconActiveColor={iconColor}
+							trackColor={BaseColors.WHITE}
+							isCurrentTrack={true}
```
[the remark this answers is not in the document]
Comment (okeinom) [reply src/components/molecules/buttonPlayCircle.tsx:60]: updated

File: src/components/molecules/buttonPlayCircle.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+					handlePlayClick();
+				}}
+				className={styles.play}
+				aria-label={intl.formatMessage({
+					id: 'playButton__playLabel',
+					defaultMessage: 'play',
+					description: 'play button play label',
+				})}
+			>
+				{isCurrentTrack ? (
```
[the remark this answers is not in the document]
Comment (okeinom) [reply src/components/molecules/buttonPlayCircle.tsx:53]: not sure what you'r talking about here...

File: src/components/atoms/playProgress.tsx [hunk 5]
```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+
+import Styles from './playProgress.module.scss';
+
+type Props = {
+	isPlaying: boolean;
+	progressPercentage?: number;
+	activeColor?: string;
```
[the remark this answers is not in the document]
Comment (okeinom) [reply src/components/atoms/playProgress.tsx:8]: updated

File: src/components/atoms/playTime.tsx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+}) => {
+	const formattedDuration = useFormattedDuration(duration);
+	const remainingDuration = useFormattedDuration(duration * (1 - progress));
+
+	return (
+		<>
+			<Heading6
+				large
+				className={clsx(styles.duration, {
+					[styles[textSecondaryColor]]: textSecondaryColor,
```
[the remark this answers is not in the document]
Comment (okeinom) [reply src/components/atoms/playTime.tsx:33]: updated

File: src/components/atoms/playTime.tsx [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+import styles from './playTime.module.scss';
+
+interface Props {
+	duration: number;
+	progress: number;
+	textSecondaryColor: BaseColors;
+	iconSecondaryColor: BaseColors;
+}
+
+const PlayTime: React.FC<Props> = ({
```
[the remark this answers is not in the document]
Comment (okeinom) [reply src/components/atoms/playTime.tsx:19]: removed

File: src/components/molecules/teaseRecording.module.scss (hunk 1 above)
Comment (okeinom) [reply src/components/molecules/teaseRecording.module.scss:67]: updated

File: src/components/molecules/buttonPlayCircle.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (alangumer) [reply src/components/molecules/buttonPlayCircle.tsx:58]: Remove the isCurrentTrack validation, and add it along with the session.progress validation in line 70, and update the properties for PlayProgress as needed.

[the remark this answers is not in the document]
Comment (okeinom) [reply src/components/molecules/buttonPlayCircle.tsx]: updated. Thanks.

File: src/components/molecules/buttonPlayCircle.tsx [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+						inactiveColor={iconColor}
+						progressPercentage={session.progress}
+						bufferedProgress={session.bufferedProgress}
+						isCurrentTrack={isCurrentTrack}
+					/>
+				) : (
+					<IconPlay color={iconColor} />
+				)}
+			</button>
+			<>
```
Comment (alangumer) [inline src/components/molecules/buttonPlayCircle.tsx:70]: Remove this.

**Review by alangumer — APPROVED** [review]

**Pull request merged**